utils/data_utils.py

```diff
@@ -12,6 +12,12 @@
 from sklearn.preprocessing import StandardScaler
 
 
+def intersperse(lst, item):
```

Comment:
Add comments for this function. For example, what is it for?

Reply:
Added, please check the latest version and let me know if there's further revision required: https://github.com/open-mmlab/Amphion/pull/131/files#diff-d8373728b78ea7f07bc4558415949336dd3efbe3ca55c04371357a5ea37895bd